### DISCUSSION TITLE
Do i need to ask for access_background_location permission  separately on android 13 and above?

### DISCUSSION THREAD

**Discussion by haziqali-2019** [HaylLtd/cordova-plugin-background-geolocation#196] [Q&A]
Without adding <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" /> in config.xml, this plugin does not seem to work on android.

Anyone has any info on it?

**Comment by HarelM** [comment]
This plugin does not need the access_background_location permission as it runs as a foreground service (which can run as a non-top-most app).
Adding this permission requires a special treatment from google which is not needed for most cases of the usage of this plugin.
If you absolutely have to add it, please add it to your config separately.
